scripts/run_boss_locally.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python3
+
+import os
+import sys
+import tempfile
+import shutil
+import glob
+import pydbus
+import shutil
+from gi.repository import Gio
+
+os.putenv("PYTHONPATH", os.path.abspath(".."))
+
+MODULES_DIR = os.path.abspath("../pyanaconda/modules")
+DBUS_SERVICES_DIR = "../data/dbus/"
+
+print("creating a temporary directory for DBUS service files")
+temp_service_dir = tempfile.TemporaryDirectory()
+print(temp_service_dir.name)
+
+print("copying service files")
+for file_path in glob.glob(DBUS_SERVICES_DIR +  "*.service"):
+    filename = os.path.split(file_path)[1]
+    target_file_path = os.path.join(temp_service_dir.name, filename)
+    shutil.copy(file_path, target_file_path)
+
+#input("lets check the service files")
+
+test_dbus = Gio.TestDBus()
+
+# set service folder
+test_dbus.add_service_dir(temp_service_dir.name)
+
+# start the custom DBUS daemon
+print("starting custom dbus session")
+test_dbus.up()
+print(test_dbus.get_bus_address())
+
+# our custom bus is now running, connect to it
+test_dbus_connection = pydbus.connect(test_dbus.get_bus_address())
+dbus_object = test_dbus_connection.get("org.freedesktop.DBus", "/org/freedesktop/DBus")
+
+print("starting Boss")
+dbus_object.StartServiceByName("org.fedoraproject.Anaconda.Boss", 42)
```

Comment:
I don't like magic numbers, could you please name this number?

Reply:
That parameter is actually unused but you have to provide a number when calling the function. But indeed, that can be a constant.